src/authentication/index.ts

```diff
@@ -26,7 +30,27 @@ export default class Authentication extends BaseApi {
     return axios.post<Credentials>(url, stringify(credentials), config);
   }
 
+  async refreshToken(): Promise<AxiosResponse> {
+    const { refreshToken } = this.getTokens();
+    const credentials = {
+      refresh_token: refreshToken,
+      client_id: this.config.clientId,
+      client_secret: this.config.clientSecret,
+      grant_type: 'refresh_token',
+    };
+
+    const url = `${this.getAccountUrl()}auth/oauth2/token`;
+    const config = {
+      headers: {
+        'Content-type': 'application/x-www-form-urlencoded',
+      },
+    };
+    return axios.post<Credentials>(url, stringify(credentials), config);
+  }
```

Comment:
The type in `post<Credentials>` is not correct, the response is not `Credentials`. I suggest we add a type for the `TokenResponse` in `authentication/types.ts` and then also adjust the return type to the proper one instead of the generic `AxiosResponse`. 
```suggestion
  async refreshToken(): Promise<AxiosResponse<TokenResponse>> {
    const { refreshToken } = this.getTokens();
    const credentials = {
      refresh_token: refreshToken,
      client_id: this.config.clientId,
      client_secret: this.config.clientSecret,
      grant_type: 'refresh_token',
    };

    const url = `${this.getAccountUrl()}auth/oauth2/token`;
    const config = {
      headers: {
        'Content-type': 'application/x-www-form-urlencoded',
      },
    };
    return axios.post<TokenResponse>(url, stringify(credentials), config);
  }
```